google/cloud/trace/v2/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "cloudtrace.googleapis.com",
+    "api_shortname": "cloudtrace",
+    "client_documentation": "https://cloud.google.com/cpp/reference/trace/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/559776",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Stackdriver Trace API",
```

Comment:
Cloud Trace API

Reply:
Fixed.  You may want to send a CL to fix the (future) source of truth for these:

https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudtrace/v1/cloudtrace_v1.yaml